License link

oorestisime opened the pull request:
* I added a link in the infobox which points at the license in the copyright BP.
* Fixed the issue with the license synopsis. The modifications now allow us to parse the synopsis splitting by the separators 'and', 'or', ',' then create links to the standard licenses and/or license text inside the License if they exist.

Example:
FSF-unlimited or GPL-2
for FSF-unlimited the link will points to the text in the license paragraphs (inside the page if such a paragraph exists) and for GPL-2 a link to opensource.org

matthieucan commented:
Awesome!
My 2 cents: I'd replace the "License" link by something more explicit, e.g. "view license information" (respecting the lowercase of the list). 
How can I test the or/and/, splitting? (Local link?)
Besides that, merge-ready :)

oorestisime commented:
You can try /copyright/license/gnubg/1.02.000-2/
there are licenses such as 
- GPL-2+ with Autoconf exception or GPL-3+ 
-  FSF-configure , and GPL-2+ with Libtool exception or GPL-3+

oorestisime commented:
I made the change for the license link (squashed and rebased with current master).
Let me know if there are more things to touch :)

matthieucan commented:
Thanks for the link! Could you add a test for that?

matthieucan commented:
Oh, and a test for the infobox link as well?

oorestisime commented:
Added the tests. I had to add the - (i.e {%- ...%}) parameter in the jinja template to avoid huge line breaks and finally be able to construct a test. (otherwise i had to add many \n\n)
I rebased the branch as well.
